buildpack.md

```diff
@@ -115,24 +115,26 @@ Executable: `/bin/build <layers[EIC]> <platform[AR]> <plan[E]>`, Working Dir: `<
-| `<layers>/launch.toml`         | App metadata (see [launch.toml](#launchtoml-toml))
-| `<layers>/store.toml`          | Persistent metadata (see [store.toml](#storetoml-toml))
-| `<layers>/<layer>.toml`        | Layer metadata (see [Layer Content Metadata](#layer-content-metadata-toml))
-| `<layers>/<layer>/bin/`        | Binaries for launch and/or subsequent buildpacks
-| `<layers>/<layer>/lib/`        | Shared libraries for launch and/or subsequent buildpacks
-| `<layers>/<layer>/profile.d/`  | Scripts sourced by Bash before launch
-| `<layers>/<layer>/include/`    | C/C++ headers for subsequent buildpacks
-| `<layers>/<layer>/pkgconfig/`  | Search path for pkg-config for subsequent buildpacks
-| `<layers>/<layer>/env/`        | Env vars for launch and/or subsequent buildpacks
-| `<layers>/<layer>/env.launch/` | Env vars for launch (after `env`, before `profile.d`)
-| `<layers>/<layer>/env.build/`  | Env vars for subsequent buildpacks (after `env`)
-| `<layers>/<layer>/*`           | Other content for launch and/or subsequent buildpacks
+| Output                                  | Description
+|-----------------------------------------|--------------------------------------
+| [exit status]                           | Success (0) or failure (1+)
+| `/dev/stdout`                           | Logs (info)
+| `/dev/stderr`                           | Logs (warnings, errors)
+| `<plan>`                                | Refinements to the [Buildpack Plan](#buildpack-plan-toml) (TOML)
+| `<layers>/launch.toml`                  | App metadata (see [launch.toml](#launchtoml-toml))
+| `<layers>/store.toml`                   | Persistent metadata (see [store.toml](#storetoml-toml))
+| `<layers>/<layer>.toml`                 | Layer metadata (see [Layer Content Metadata](#layer-content-metadata-toml))
+| `<layers>/<layer>/bin/`                 | Binaries for launch and/or subsequent buildpacks
+| `<layers>/<layer>/lib/`                 | Shared libraries for launch and/or subsequent buildpacks
+| `<layers>/<layer>/profile.d/`           | Scripts sourced by Bash before launch
+| `<layers>/<layer>/profile.d/<process>`  | Scripts sourced by Bash before launch for the launched process
+| `<layers>/<layer>/include/`             | C/C++ headers for subsequent buildpacks
+| `<layers>/<layer>/pkgconfig/`           | Search path for pkg-config for subsequent buildpacks
+| `<layers>/<layer>/env/`                 | Env vars for launch and/or subsequent buildpacks
+| `<layers>/<layer>/env.launch/`          | Env vars for launch (after `env`, before `profile.d`)
+| `<layers>/<layer>/env.launch/<process>` | Env vars for launch (after `env`, before `profile.d`) for the launched process
+| `<layers>/<layer>/env.build/`           | Env vars for subsequent buildpacks (after `env`)
+| `<layers>/<layer>/*`                    | Other content for launch and/or subsequent buildpacks
```

Comment:
I think the trailing slash is needed here to make it clearer that a file in `profile.d` that happens to be named after `<process>` isn't process-specific.

```suggestion
| Output                                   | Description
|------------------------------------------|--------------------------------------
| [exit status]                            | Success (0) or failure (1+)
| `/dev/stdout`                            | Logs (info)
| `/dev/stderr`                            | Logs (warnings, errors)
| `<plan>`                                 | Refinements to the [Buildpack Plan](#buildpack-plan-toml) (TOML)
| `<layers>/launch.toml`                   | App metadata (see [launch.toml](#launchtoml-toml))
| `<layers>/store.toml`                    | Persistent metadata (see [store.toml](#storetoml-toml))
| `<layers>/<layer>.toml`                  | Layer metadata (see [Layer Content Metadata](#layer-content-metadata-toml))
| `<layers>/<layer>/bin/`                  | Binaries for launch and/or subsequent buildpacks
| `<layers>/<layer>/lib/`                  | Shared libraries for launch and/or subsequent buildpacks
| `<layers>/<layer>/profile.d/`            | Scripts sourced by Bash before launch
| `<layers>/<layer>/profile.d/<process>/`  | Scripts sourced by Bash before launch for the launched process
| `<layers>/<layer>/include/`              | C/C++ headers for subsequent buildpacks
| `<layers>/<layer>/pkgconfig/`            | Search path for pkg-config for subsequent buildpacks
| `<layers>/<layer>/env/`                  | Env vars for launch and/or subsequent buildpacks
| `<layers>/<layer>/env.launch/`           | Env vars for launch (after `env`, before `profile.d`)
| `<layers>/<layer>/env.launch/<process>/` | Env vars for launch (after `env`, before `profile.d`) for the launched process
| `<layers>/<layer>/env.build/`            | Env vars for subsequent buildpacks (after `env`)
| `<layers>/<layer>/*`                     | Other content for launch and/or subsequent buildpacks
```